### PR TITLE
Add kotlin 1.5.30

### PR DESCRIPTION
**Pull request by supergrecko**
Compilers for Kotlin 1.5.30

Infra: https://github.com/compiler-explorer/infra/pull/575

### CODE REVIEW

File: etc/config/kotlin.amazon.properties [hunk 1]
```diff
@@ -60,3 +60,7 @@ compiler.kotlinc1521.exe=/opt/compiler-explorer/kotlin-jvm-1.5.21/bin/kotlinc-jv
 compiler.kotlinc1521.semver=1.5.21
 compiler.kotlinc1521.java_home=/opt/compiler-explorer/jdk-16.0.1
 compiler.kotlinc1521.runtime=/opt/compiler-explorer/jdk-16.0.1/bin/java
+compiler.kotlinc1521.exe=/opt/compiler-explorer/kotlin-jvm-1.5.30/bin/kotlinc-jvm
```
Comment (RubenRBS) [inline etc/config/kotlin.amazon.properties:63]:
```suggestion
compiler.kotlinc1530.exe=/opt/compiler-explorer/kotlin-jvm-1.5.30/bin/kotlinc-jvm
```

**Review by RubenRBS — APPROVED** [review]

**Comment by RubenRBS** [comment]
Do we want to bump the default compiler to the newer one too? Or should that stay at 1.5.00?

**Comment by supergrecko** [comment]
> Do we want to bump the default compiler to the newer one too? Or should that stay at 1.5.00?

I'll bump it, thanks for the reminder!

**Review by mattgodbolt — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**